test/e2e/storage/vsphere/vsphere_common.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vsphere
+
+import "os"
+
+var (
+	SPBMPolicyName = os.Getenv("VSPHERE_SPBM_POLICY_NAME")
+	StorageClassDatastoreName = os.Getenv("VSPHERE_DATASTORE")
+	SecondSharedDatastore = os.Getenv("VSPHERE_SECOND_SHARED_DATASTORE")
+	KubernetesClusterName = os.Getenv("VSPHERE_KUBERNETES_CLUSTER")
+	SPBMTagPolicy = os.Getenv("VSPHERE_SPBM_TAG_POLICY")
+)
+
+
+var (
```

Comment:
can all of these vars be made `const`s?

Reply:
This sounds good to me. Made env var const.